test/system/projects/samples_test.rb

```diff
@@ -41,1291 +199,1535 @@ class SamplesTest < ApplicationSystemTestCase
       assert_text I18n.t(:'action_policy.policy.project.sample_listing?', name: @project.name)
     end
 
-    test 'should create sample' do
+    test 'create sample' do
+      ### SETUP START ###
       visit namespace_project_samples_url(@namespace, @project)
-      assert_selector 'a', text: I18n.t('projects.samples.index.new_button'), count: 1
+      within('#samples-table table tbody') do
+        # sample does not currently exist
+        assert_no_text 'New Name'
```

Comment:
**nitpick (blocking):** This could pass even if a sample with `New Name` did exist in the project, since there is no assert to ensure that the samples table has been loaded.

Reply:
Added in d60b9028eeca99d5aad54b37f1e7ce872ceb8dac